lms/djangoapps/courseware/date_summary.py

```diff
@@ -204,10 +204,20 @@ class CertificateAvailableDate(DateSummary):
     title = ugettext_lazy('Certificate Available')
 
     @property
+    def active_certificates(self):
+        return [
+            certificate for certificate in self.course.certificates.get('certificates', [])
+            if certificate.get('is_active', False)
```

Comment:
Wait ... active_certificates returns a list of certificates where 'is_active' is False?

Reply:
@ssemenova It's a little unclear. It returns false if **not** 'is_active'.